### ISSUE TITLE
Error yii\httpclient\XmlFormatter::itemTag

### ISSUE THREAD

**Issue by Abylaikhan** [yiisoft/yii2-httpclient#45]
I create query with joins, then send(post) via http-client(xml), I getting error: "Getting unknown property: yii\httpclient\XmlFormatter::itemTag" like this.

```php
$query = User::find()
                    ->with('orders')
                    ->where(['id' => $user_id['id']])
                    ->asArray()
                    ->one();
                $client = new Client([
                    'transport' => 'yii\httpclient\CurlTransport',
                    'requestConfig' => [
                        'format' => Client::FORMAT_XML
                    ],
                    'responseConfig' => [
                        'format' => Client::FORMAT_XML
                    ],
                ]);
                $response = $client->createRequest()
                    ->setMethod('post')
                    ->setUrl('http://site.com/orders/own')
                    ->setData($query)
                    ->send();
```
How to solve it?

**Comment by klimov-paul** [comment]
@Abylaikhan, please use <a href="http://www.yiiframework.com/forum/index.php/forum/42-general-discussions-for-yii-20/">Official Yii 2 forum</a> to ask a questions - GitHub is a bug tracking system.

**Comment by cebe** [comment]
@klimov-paul https://github.com/yiisoft/yii2-httpclient/blob/master/XmlFormatter.php#L99 $this->itemTag  does not exist as far as I see.

**Comment by klimov-paul** [comment]
Why it should exist?

**Comment by cebe** [comment]
because the code tries to access it? :)

**Comment by klimov-paul** [comment]
which code?

**Comment by klimov-paul** [comment]
OK, I see

**Comment by klimov-paul** [comment]
https://github.com/yiisoft/yii2-httpclient/blob/master/XmlFormatter.php#L99

**Comment by Abylaikhan** [comment]
Cebe How to solve it? I don`t understand this error?

**Comment by cebe** [comment]
it is a bug in the httpclient implementation.

**Comment by Abylaikhan** [comment]
thanks Cebe

**Comment by klimov-paul** [comment]
Resolved by commit 78c809277bf98a324531522cdb858107842d0819
